### ISSUE TITLE
3.3.9 build fail , theme-default version detect error?

### ISSUE THREAD

**Issue by tsuyoshicho** [githon/githon#42]
Report:

Try convert from gitbook to githon.
https://github.com/tsuyoshicho/WhenScarletIceMelted/tree/feature/githon

Setup and `npm install` success for `githon 3.3.9` and others.

But, when `npm run build`(githon build) phase, error occur.

Platform: Windows, Linux
Node: v14.4.0

<details>
<summary>`Githon doesn't satisfy the requirements of this plugin: >=4.0.0-alpha`</summary>

```
WhenScarletIceMelted on  feature/githon via ⬢ v14.4.0 took 15s
⏰[ 2020-06-20T07:40:09 ] ❯ npm install

> node-sass@4.14.1 install D:\Data\ReposWork\git\WhenScarletIceMelted\node_modules\node-sass
> node scripts/install.js

Cached binary found at C:\Users\tsuyoshi\AppData\Roaming\npm-cache\node-sass\4.14.1\win32-x64-83_binding.node

> node-sass@4.14.1 postinstall D:\Data\ReposWork\git\WhenScarletIceMelted\node_modules\node-sass
> node scripts/build.js

Binary found at D:\Data\ReposWork\git\WhenScarletIceMelted\node_modules\node-sass\vendor\win32-x64-83\binding.node
Testing binary
Binary is fine
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

added 788 packages from 768 contributors and audited 790 packages in 378.876s

60 packages are looking for funding
  run `npm fund` for details

found 3 vulnerabilities (1 low, 2 high)
  run `npm audit fix` to fix them, or `npm audit` for details
WhenScarletIceMelted on  feature/githon via ⬢ v14.4.0 took 6m20s
⏰[ 2020-06-20T07:46:34 ] ❯ npm run build

> WhenScarletIceMelted@1.0.0 build D:\Data\ReposWork\git\WhenScarletIceMelted
> githon build

info: 9 plugins are installed
info: 9 explicitly listed
info: plugin "creativecommons is loaded
info: plugin "styles-sass-fix is loaded
info: plugin "puml is loaded
info: plugin "highlight is loaded
info: plugin "search is loaded
info: plugin "lunr is loaded
info: plugin "fontsettings is loaded
info: plugin "theme-default is loaded
Githon doesn't satisfy the requirements of this plugin: >=4.0.0-alpha
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! WhenScarletIceMelted@1.0.0 build: `githon build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the WhenScarletIceMelted@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\tsuyoshi\AppData\Roaming\npm-cache\_logs\2020-06-19T22_46_47_721Z-debug.log
exit status 1
WhenScarletIceMelted on  feature/githon via ⬢ v14.4.0 took 2s
⏰[ 2020-06-20T07:46:47 ] ❯
```
</details>

**Comment by azu** [comment]
Probably, this error make confuse.
It looks like that the next plugin of `theme-default` throw an Error.

Can you paste your `book.js`?

**Comment by azu** [comment]
Ah, OK. https://github.com/tsuyoshicho/WhenScarletIceMelted/blob/feature/githon/book.json#L7

**Comment by azu** [comment]
https://github.com/GitbookIO/plugin-disqus/blob/e77de0febf42faa075c0d21e1e6e2e82bdc4f3f2/package.json#L10 cause this error.

>  "gitbook": ">=4.0.0-alpha"

This behavior is correct because GitBook 4.0.0 does not exist...
But, we need to relux this validation.

**Comment by azu** [comment]
I've fixed this issue and released [v3.3.10](https://github.com/githon/githon/releases/tag/v3.3.10).

Please try it .

```
npm i githon@latest --save-dev
```

**Comment by tsuyoshicho** [comment]
Thanks fixing!!
